src/lib.rs

```diff
@@ -1,54 +1,76 @@
+use std::sync::Once;
+
+#[cfg(feature = "tracing")]
+use tracing::Level;
+
+#[macro_use]
```

Comment:
I don't think you need `macro_use`, do you ?

Reply:
You do, otherwise the macro's won't be in scope. it allows not exporting them from datachannel as a crate, but they are available privately at the crate level.

Reply:
This relates to my other comment, if you use `use crate::logger; logger::info!(...)` in the code then you don't need [`macro_use`](https://doc.rust-lang.org/reference/macros-by-example.html#the-macro_use-attribute) which simply brings macros out of their original module within the current crate.